index.html

```diff
@@ -30,16 +48,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div id="b1" class="box">Box 1</div>
```

Comment:
Using ID's feels like cheating, but if it achieves the goal I guess we did not say you could not use them.
Consider this method also: https://www.w3schools.com/cssref/sel_nth-child.asp